tests/common/devices/sonic.py

```diff
@@ -32,13 +32,14 @@ class SonicHost(AnsibleHostBase):
     This type of host contains information about the SONiC device (device info, services, etc.),
     and also provides the ability to run Ansible modules on the SONiC device.
     """
-    DEFAULT_ASIC_SERVICES = ["bgp", "database", "lldp", "swss", "syncd", "teamd"]
 
     def __init__(self, ansible_adhoc, hostname,
                  shell_user=None, shell_passwd=None,
                  ssh_user=None, ssh_passwd=None):
         AnsibleHostBase.__init__(self, ansible_adhoc, hostname)
 
+        self.DEFAULT_ASIC_SERVICES = ["bgp", "database", "lldp", "swss", "syncd", "teamd", "macsec"]
```

Comment:
Need to check for the dut type as SpineRouter and add macsec as default asic service -- else would break for non T2.

Reply:
@judyjoseph , Updated the code to add macsec for dut type as SpineRouter. Please take a look. Verified by running 'Container-checker' and 'Autorestart' test cases and they worked fine for macsec feature.